google/ggrc-core
# GGRC-4753,GGRC-4444 Make related_sources and destinations read-only

# Issue description

If the FE PUTs empty `related_sources` or `related_destinations`, actual relationships get deleted, even the ones we rely on heavily, like Audit to Assessment.

# Steps to test the changes

Steps to reproduce:
1. Open the same audit in tab 1 and tab 2
2. Create an Assessment in tab 2
3. Return to tab 1 and update ACL via inline edit, e.g. add/change Auditor
4. Return to tab 2 and refresh the page: Assessment is unmapped from Audit and is not shown in tree view
*Actual Result:* Mapping to Audit was deleted for Assessment when Audit is opened in two tabs and user makes update of ACL w/o refreshing a page
*Expected Result:* Mapping to Audit should not be deleted for Assessment

# Solution description

Same as with #7502, just make the relationship a read-only attribute.

# Sanity checklist




# PR Review checklist
